packages/ui/src/theme/css/component/radiogroup.scss

```diff
@@ -1,4 +1,5 @@
 .amplify-radiogroup {
   gap: inherit;
   flex-direction: inherit;
+  align-items: flex-start;
```

Comment:
nit: since the parent field container has set `align-items: flex-start;`, would it make sense to set it `inherit` to be consistent with other props there?

Reply:
Oh yea, I was playing around with align items at different places and I think you are right.